_includes/master/ipnat.md

```diff
@@ -0,0 +1,6 @@
+IPNAT contains a single NAT mapping for a WorkloadEndpoint resource.
```

Comment:
Why is this capitalized - IPNAT?

Reply:
1. It's an acronym, not a yaml/json field name so I've kept it capitalized
2. I'm following the same style as [EndpointPort](https://docs.projectcalico.org/master/reference/calicoctl/resources/workloadendpoint#endpointport) where the casing is same as the field name

Reply:
I suppose so.

I was originally thinking this was something that existed in our API, but I guess it doesn't actually show up in the yaml so this is fine.